cli/exp_task_list.go

```diff
@@ -16,20 +16,29 @@ import (
 type taskListRow struct {
 	Task codersdk.Task `table:"t,recursive_inline"`
 
+	OwnerAndName    string `table:"name,default_sort"`
 	StateChangedAgo string `table:"state changed"`
+	Healthy         bool   `json:"-" table:"healthy"`
```

Comment:
drive-by: I consolidated on `taskListRow` for `task list` and `task status`, so had to add this in here.